src/pages/GenerateKeys/index.tsx

```diff
@@ -219,14 +219,17 @@ const _GenerateKeysPage = ({
         <Text className="mb20">
           <FormattedMessage
             defaultMessage="You may choose to provide a withdrawal address with your initial
-            deposit to automatically enable reward payments and also the ability
-            to fully exit your funds anytime after the Shanghai/Capella upgrade.
-            This address should be to a regular Ethereum address and will be the
-            only address funds can be sent to from your new validator accounts."
+            deposit to automatically enable reward payments and also the ability to fully
+            exit your funds at anytime (recommended). This address should be to a regular
+            Ethereum address and will be the only address funds can be sent to from your new
+            validator accounts, and cannot be changed once chosen."
```

Comment:
"and cannot be changed once chosen."

This feels buried here. Seems like very important information that deserves its own sentence at least. I'm only commenting, so this might already be solved by the page(s) context.